iset.mm

```diff
@@ -69804,6 +69832,170 @@ property of disjoint unions (see ~ updjud in the case of functions).
+    nnnninfeq2.n $e |- ( ph -> N e. _om ) $.
+    nnnninfeq2.1 $e |- ( ph -> ( P ` U. N ) = 1o ) $.
+    nnnninfeq2.0 $e |- ( ph -> ( P ` N ) = (/) ) $.
+    $( Mapping of a natural number to an element of ` NN+oo ` .  Similar to
+       ~ nnnninfeq but if we have information about a single ` 1o ` digit, that
+       gives information about all previous digits.  (Contributed by Jim
+       Kingdon, 4-Aug-2022.) $)
+    nnnninfeq2 $p |- ( ph
+        -> P = ( i e. _om |-> if ( i e. N , 1o , (/) ) ) ) $=
+      ( vx vj cuni cfv c1o wceq wral com wcel wa wi c0 vw vk vf cv adantr unieq
+      csuc fveqeq2d anbi2d raleq imbi12d ral0 a1i csn wn uni0 id 3eqtr4a fveq2d
+      cun wtr word nnord ordtr ad3antlr wb unisucg mpbid simpr eqtr3d sylan9eqr
+      syl wss c2o xnninf nninff nnpredcl adantl ffvelrnd el2oss1o ad3antrrr wne
+      wf simp-4r nnsucpred syl2anc eqtrd suceq fveq2 sseq12d cmap fveq1 ralbidv
+      neqned co df-nninf elrab2 sylib simprd rspcdva eqsstrrd eqssd wdc nndceq0
+      wo exmiddc mpjaodan simplr fveqeq2 ralunsn mpbir2and df-suc raleqi sylibr
+      mpd exp31 expcom a2d impexp 3imtr4g finds mpcom mpdan nnnninfeq ) AIBCDEF
+      ADKZBLMNZIUDZBLMNZIDOZGDPQZAYFRZYIAYJYFFUEAUAUDZKZBLMNZRZYHIYLOZSATKZBLMN
+      ZRZYHITOZSAUBUDZKZBLZMNZRZYHIUUAOZSZAUUAUGZKZBLZMNZRZYHIUUHOZSZYKYISUAUBD
+      YLTNZYOYSYPYTUUOYNYRAUUOYMYQMBYLTUFUHUIYHIYLTUJUKYLUUANZYOUUEYPUUFUUPYNUU
+      DAUUPYMUUBMBYLUUAUFUHUIYHIYLUUAUJUKYLUUHNZYOUULYPUUMUUQYNUUKAUUQYMUUIMBYL
+      UUHUFUHUIYHIYLUUHUJUKYLDNZYOYKYPYIUURYNYFAUURYMYEMBYLDUFUHUIYHIYLDUJUKYTY
+      SYHIULUMUUAPQZAUUDUUFSZSAUUKUUMSZSUUGUUNUUSAUUTUVAAUUSUUTUVASAUUSRZUUTUUK
+      UUMUVBUUTRZUUKRZYHIUUAUUAUNUTZOZUUMUVDUVFUUFUUABLZMNZUVDUUDUUFUVDUUATNZUU
+      DUVIUOZUVIUVDUUCUVGMUVIUUBUUABUVIYQTUUBUUAUPUUATUFUVIUQURUSUVDUUJUVGMUVDU
+      UIUUABUVDUUAVAZUUIUUANZUUSUVKAUUTUUKUUSUUAVBUVKUUAVCUUAVDVLVEUUSUVKUVLVFA
+      UUTUUKUUAPVGVEVHUSUVCUUKVIVJZVKUVDUVJRZUUCMUVBUUCMVMZUUTUUKUVJUVBUUCVNQUV
+      OUVBPVNUUBBAPVNBWCZUUSABVOQZUVPEBVPVLUEUUSUUBPQZAUUAVQZVRVSUUCVTVLWAUVNMU
+      UBUGZBLZUUCUVNUWAUVGMUVNUVTUUABUVNUUSUUATWBUVTUUANAUUSUUTUUKUVJWDUVNUUATU
+      VDUVJVIWNUUAWEWFUSUVDUVHUVJUVMUEWGUVDUWAUUCVMZUVJUVDJUDZUGZBLZUWCBLZVMZUW
+      BJPUUBUWCUUBNZUWEUWAUWFUUCUWHUWDUVTBUWCUUBWHUSUWCUUBBWIWJAUWGJPOZUUSUUTUU
+      KABVNPWKWOZQZUWIAUVQUWKUWIREUWDUCUDZLZUWCUWLLZVMZJPOUWIUCBUWJVOUWLBNZUWOU
+      WGJPUWPUWMUWEUWNUWFUWDUWLBWLUWCUWLBWLWJWMUCJWPWQWRWSWAUUSUVRAUUTUUKUVSVEW
+      TUEXAXBUUSUVIUVJXEZAUUTUUKUUSUVIXCUWQUUAXDUVIXFVLVEXGUVBUUTUUKXHXOUVMUUSU
+      VFUUFUVHRVFAUUTUUKYHUVHIUUAUUAPYGUUAMBXIXJVEXKYHIUUHUVEUUAXLXMXNXPXQXRAUU
+      DUUFXSAUUKUUMXSXTYAYBYCHYD $.
+  $}
+
+  ${
+    nninfisol.x $e |- ( ph -> X e. NN+oo ) $.
+    nninfisol.0 $e |- ( ph -> ( X ` N ) = (/) ) $.
+    nninfisol.n $e |- ( ph -> N e. _om ) $.
+
+    ${
+      $d N i $.  $d N j $.  $d X j $.  $d i ph $.
+      nninfisollem0.0 $e |- ( ph -> N = (/) ) $.
+      $( Lemma for ~ nninfisol .  The case where N is zero.  (Contributed by
```

Comment:
I was wondering about this too. Upon looking at it more closely, the backticks produce italics and a different color, so that's a pretty significant difference (and better with the backticks).

Changed.